Products/CMFPlone/resources/browser/resource.py

```diff
@@ -236,8 +259,29 @@ def check_dependencies(bundle_name, depends, bundles):
                 rel="stylesheet",
                 **{"data-bundle": "diazo"},
             )
+            last_css_bundle = "theme"
+
+        # Fix "all" dependencies to be rendered after the last known non-all resource.
+        for resource in depends_all_js:
+            # Change all "all" to last known JS resource
+            dependencies = map(
+                lambda dep: dep if dep != "all" else last_js_bundle, resource.depends
+            )
+            # Remove None values, e.g. if no known JS resource was found
+            dependencies = filter(lambda dep: dep is not None, dependencies)
+            resource.depends = list(dependencies)
+
+        for resource in depends_all_css:
+            # Change all "all" to last known JS resource
+            dependencies = map(
+                lambda dep: dep if dep != "all" else last_css_bundle, resource.depends
+            )
+            # Remove None values, e.g. if no known JS resource was found
```

Comment:
> Remove None values, e.g. if no known JS resource was found

This is for CSS right?
The code in this cycle and in the one above looks the same.
If the treatment for the resources is equal maybe it is better to move it to a method?